*** cdtray.py	2013-05-18 12:22:44.124597662 +0200
--- cdtray_patch.py	2013-05-18 12:40:37.565373880 +0200
***************
*** 11,17 ****
  import gst
  
  import ConfigParser
! from os import environ, getcwd, walk
  from os.path import isfile, join
  from commands import getoutput
  from ctypes import CDLL
--- 11,17 ----
  import gst
  
  import ConfigParser
! from os import environ, walk
  from os.path import isfile, join
  from commands import getoutput
  from ctypes import CDLL
***************
*** 20,26 ****
  
  configfile = environ['HOME']+'/.cdtray'
  
! t = gettext.translation('cdtray', 'lang')
  _ = t.ugettext
  
  class CDTRAY():
--- 20,26 ----
  
  configfile = environ['HOME']+'/.cdtray'
  
! t = gettext.translation('cdtray', '/usr/share/locale')
  _ = t.ugettext
  
  class CDTRAY():
***************
*** 42,48 ****
  
  		self.statusicon = gtk.StatusIcon()
  
! 		self.statusicon.set_from_file('cdtray.svg')
  		self.statusicon.connect("popup-menu", self.show_menu)
  		self.statusicon.connect('activate', self.play)
  
--- 42,48 ----
  
  		self.statusicon = gtk.StatusIcon()
  
! 		self.statusicon.set_from_icon_name('cdtray')
  		self.statusicon.connect("popup-menu", self.show_menu)
  		self.statusicon.connect('activate', self.play)
  
***************
*** 223,230 ****
  		# Send notify
  		if self.shownotify == 1:
  			pynotify.init('CD Tray')
! 			img = '%s/cdtray.svg' % getcwd()
! 			notify = pynotify.Notification('CD Tray', _('Playing track %s') % self.actual_track, img)
  			notify.show()
  
  	def update_jt_menu(self):
--- 223,229 ----
  		# Send notify
  		if self.shownotify == 1:
  			pynotify.init('CD Tray')
! 			notify = pynotify.Notification('CD Tray', _('Playing track %s') % self.actual_track, '/usr/share/icons/cdtray.svg')
  			notify.show()
  
  	def update_jt_menu(self):
***************
*** 398,404 ****
  		f.close()
  		info.set_comments(_('Special Thanks:\nTo the Desdelinux users for supporting me with their ideas\nTo Desdelinux\'s user proper for helping me with the name of the program\nAnd for all of you for downloading and using this program'))
  		info.set_website('https://github.com/son-link/CD-Tray')
! 		info.set_website_label(_("Proyect page"))
  		info.set_translator_credits('English: Alfonso Saavedra "Son Link"\nAurosZx')
  		def close(w, res):
  			w.hide()
--- 397,403 ----
  		f.close()
  		info.set_comments(_('Special Thanks:\nTo the Desdelinux users for supporting me with their ideas\nTo Desdelinux\'s user proper for helping me with the name of the program\nAnd for all of you for downloading and using this program'))
  		info.set_website('https://github.com/son-link/CD-Tray')
! 		info.set_website_label(_("Proyect page."))
  		info.set_translator_credits('English: Alfonso Saavedra "Son Link"\nAurosZx')
  		def close(w, res):
  			w.hide()
